blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineRunImpl.java

```diff
@@ -157,6 +159,16 @@ static void clearCache() {
         PIPELINE_NODE_CONTAINER_LOADING_CACHE.invalidateAll();
     }
 
+
+    public static WorkflowRun findRun(String externalizableId) {
+        String[] tree = externalizableId.split("#");
+        Item item = Jenkins.get().getItem(tree[0], Jenkins.get());
+        if (item instanceof WorkflowJob) {
+            return ((WorkflowJob)item).getBuild(tree[1]);
```

Comment:
Just use `Run.fromExternalizableId`.

Reply:
argh yes I couldn;t find this static method